Samples/DirectMLSuperResolution/DirectMLSuperResolution.vcxproj.filters

```diff
@@ -149,5 +146,8 @@
     <CopyFileToFolders Include="Media\Videos\Turn10\FH3_540p60.mp4">
       <Filter>Assets</Filter>
     </CopyFileToFolders>
+    <CopyFileToFolders Include="Assets\weights.bin">
```

Comment:
Are you sure this does what you expect? IIRC .filters files don't participate in the build. These files should already have the "CopyFileToFolders" set in the vcxproj itself.